spec/triggers/stan.md

```diff
@@ -0,0 +1,78 @@
+# How to test the Nats streaming scaler
+
+## First start a nats streaming server with monitoring endpoint.
+
+
+Get the chart from [here](https://github.com/balchua/gonuts/tree/master/natss-chart)
+
+Execute the command:
+
+`helm install --namespace stan -n stan . `
+
+You should have the following resources in the `stan` namespace
+
+```
+kubectl -n stan get all
+NAME                 READY   STATUS    RESTARTS   AGE
+pod/stan-nats-ss-0   1/1     Running   11         37h
+
+NAME                   TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)                         AGE
+service/stan-nats-ss   NodePort   10.152.183.67   <none>        4222:31592/TCP,8222:32484/TCP   37h
+
+NAME                            READY   AGE
+statefulset.apps/stan-nats-ss   1/1     37h
+
+```
+
+Take note of the service name `stan-nats-ss`, you will use this to populate the `ScaledObject`
+
+You should also enable the monitoring endpoint of nats-streaming.
+
+## Start a Nats Streaming publisher
+
+See the example [publisher](https://github.com/balchua/gonuts/tree/master/pub) code.
+
+## Start a Nats Streaming consumer
+
+See the example [consumer](https://github.com/balchua/gonuts/tree/master/sub) code.
+
+
+## Apply `stan_scaledobject.yaml`
```

Comment:
Thanks for adding this, can you align the approach a bit with the others please?

Example: https://github.com/kedacore/keda/blob/master/spec/triggers/azure-service-bus.md

Reply:
Thanks for the review.  Can samples go to `keda/samples`?

Reply:
If you have a look at the example you'll see that only an example of the trigger itself is required here, full examples should go to /examples like you did in this PR